include stdint.h for getting uintptr_t definition

fixes do_compile_ptest_base
tests/fanotify_mark.c:191:23: error: 'uintptr_t' undeclared (first use in this function); did you mean 'intptr_t'?

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/tests/fanotify_mark.c
+++ b/tests/fanotify_mark.c
@@ -18,6 +18,7 @@
 
 # include <limits.h>
 # include <stdio.h>
+# include <stdint.h>
 # include <unistd.h>
 # include <sys/fanotify.h>
 
